clients/clearly-defined/src/main/kotlin/ClearlyDefinedService.kt

```diff
@@ -76,21 +76,27 @@ interface ClearlyDefinedService {
     /**
      * See https://github.com/clearlydefined/service/blob/661934a/schemas/swagger.yaml#L8-L14.
      */
-    enum class Server(val url: String) {
+    enum class Server(val apiUrl: String, val webUrl: String? = null, val contributionUrl: String? = null) {
         /**
-         * The ClearlyDefined production server. When submitting curations, this will create PRs against the repository
-         * at https://github.com/clearlydefined/curated-data.
+         * The production server.
          */
-        PRODUCTION("https://api.clearlydefined.io"),
+        PRODUCTION(
+            "https://api.clearlydefined.io",
+            "https://clearlydefined.io",
+            "https://github.com/clearlydefined/curated-data"
+        ),
 
         /**
-         * The ClearlyDefined development server. When submitting curations, this will create PRs against the repository
-         * at https://github.com/clearlydefined/curated-data-dev.
+         * The development server.
          */
-        DEVELOPMENT("https://dev-api.clearlydefined.io"),
+        DEVELOPMENT(
+            "https://dev-api.clearlydefined.io",
+            "https://dev.clearlydefined.io",
+            "https://github.com/clearlydefined/curated-data-dev"
+        ),
 
         /**
-         * The ClearlyDefined server when running locally.
+         * The server when running locally.
```

Comment:
Should LOCAL use empty strings rather than null values. E.g. I wonder if the caller of local expects a fallback to PRODUCTION.

Reply:
I believe these environments shouldn't be mixed, and there should be no fallback from / combination of LOCAL and PRODUCTION URLs. With a non-nullable value there's a risk that it's used as-is as the semantics of an empty string to mean "undefined" are a bit unclear. IMO `null` is a better value value for a string / URL if you want to express "undefined".

Reply:
I see. I just had blindly assumed null does not mean "undefined" but it does mean "fall back to default" as it does so in other places of the client server code.